### PR TITLE
Add stat api reference

### PR DESCRIPTION
**Pull request by AndreiKingsley**
part of #316

### CODE REVIEW

**Review by zaleslaw — APPROVED** [review]
@AndreiKingsley it's very difficult to review these changes because of tons of HTML changes, but I did not find something wrong reading the texts

File: docs/topics/apiRef/Area-API.md [hunk 1]
```diff
@@ -2,45 +2,25 @@
...
-</li>
-<li>
-<a href="#fillcolor"><format style="bold" color="DarkGray">fillColor</format></a> <format style="superscript">NonPositional</format>
-<include from="properties.topic" element-id="signature-of-nonpos-color"></include>
-</li>
-<li>
-<include from="properties.topic" element-id="signature-of-borderLine"/>
-</li>
-</list>
+<include from = "interfaces.topic" element-id="interface-of-area"></include>
```
Comment (devcrocod) [inline docs/topics/apiRef/Area-API.md:5]: Why did you do this?

**Review by devcrocod — CHANGES_REQUESTED** [review]
I don't like that you took out the signatures.
What should be taken out is what is reused, and these things are unique for each layer. Thus, you have made it difficult to maintain documentation for these layers.

File: docs/topics/apiRef/Area-API.md (hunk 1 above)
Comment (AndreiKingsley) [reply docs/topics/apiRef/Area-API.md:5]: I reused it in https://github.com/Kotlin/kandy/blob/4016c1f876b2de229e2cbe553564a18a83f03137/docs/topics/apiRef/DensityPlot-API.md

File: docs/topics/apiRef/Area-API.md [hunk 2]
```diff
@@ -2,45 +2,25 @@
...
 <format style="italic">}</format>
 </tldr>
 
 ## Properties
 
-### x
+### x {id="x_area"}
```
[the remark this answers is not in the document]
Comment (AndreiKingsley) [reply docs/topics/apiRef/Area-API.md:11]: In statistic layers you have both `x` as argument and `x` as property. Sometimes in the topic there're several layers are described (see [boxplot](https://github.com/Kotlin/kandy/blob/4016c1f876b2de229e2cbe553564a18a83f03137/docs/topics/apiRef/Boxplot-API.md)).

[the remark this answers is not in the document]
Comment (devcrocod) [reply docs/topics/apiRef/Area-API.md:11]: it's area

File: docs/topics/apiRef/Area-API.md (hunk 1 above)
Comment (devcrocod) [reply docs/topics/apiRef/Area-API.md:5]: How? The naming doesn't match there?

Comment (AndreiKingsley) [reply docs/topics/apiRef/Area-API.md:5]: What do you mean? There's include:
https://github.com/Kotlin/kandy/blob/4016c1f876b2de229e2cbe553564a18a83f03137/docs/topics/apiRef/DensityPlot-API.md?plain=1#L17

File: docs/topics/apiRef/Area-API.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (AndreiKingsley) [reply docs/topics/apiRef/Area-API.md:11]: yes, but in interfaces we should separate aesthetics references. (like in densityPlot there's `# x` argument and `# x` area aes)

**Comment by devcrocod** [comment]
I’ll render it locally and see in more detail what it looks like

@zaleslaw It’s better to watch this kind of PR in idea with the writerside plugin, then you can watch the preview. Otherwise, it’s really difficult to review xml manually

**Review by devcrocod — APPROVED** [review]

**Pull request merged**